### DISCUSSION TITLE
How to get Godmode in debug build?

### DISCUSSION THREAD

**Discussion by rabbyzero** [diasurgical/DevilutionX#7361] [General]
I build with VS 2022 and selected X64-Debug to build.
There is no Chat function in single player mode. Even in multiple player mode and ENTER GOD in chat. I cannot start godmod. I tried start with +god startup parameter. No good.

Anything I am doing wrong?

**Comment by qndel** [comment]
we aren't using chat for debug commands anymore, there's a console accessible with \`

**Reply by rabbyzero** [reply]
Thanks for the direction. I tried the console. When I input "god" or "help", it says "nil". Console is good. It has autocomplete. Let me see what it offers.

**Reply by glebm** [reply]
The debug options all start with `dev.` and then you can pick from the autocomplete menu there
